modules/about-developer-perspective.adoc

```diff
@@ -27,3 +27,8 @@ The *Developer* perspective provides workflows specific to developer use cases,
 * Group components within an application and connect the components within and across applications.
 * Integrate serverless capabilities (Technology Preview).
 * Create workspaces to edit your application code using Eclipse Che.
+
+You can use the *Topology* view to display the list of all the projects in your cluster. If you have no workloads or applications in the project, the *Topology* view displays the available options to create applications. If you have existing workloads, the *Topology* view graphically displays your workload nodes.
```

Comment:
`You can use the *Topology* view to display the list of all the projects in your cluster`

this doesn't seem very right, topology view is meant to list all the applications/workloads

Reply:
@invincibleJai, this section's content is not created but only moved from Accessing the Developer Perspective to About Developer Perspective. Are you sure if you want to change this? If yes, I can rephrase the first line.

Reply:
@shreyasiddhartha can you point me to that?

Reply:
Agree with Jai, also if its moved we should improve this anyway. The topology view is about showing workloads.

You want to change the first sentence to

```suggestion
You can use the *Topology* view to display applications, components and workloads of your project. If you have no workloads in the project, the *Topology* view will show some links to create or import them. You can also use the *Quick Search* to import components directly.
```

Reply:
yeah above suggestion looks good technically and depending on the context can add if no namespace/project is selected and the user goes to the developer perspective then the topology view shows the list of namespaces/projects if any exist.

Reply:
Thanks @jerolimov. The above suggestion looks good to me as well. I'm adding an additional serial comma before "and" in the suggested changes.